src/test/java/org/folio/rest/api/ItemDamagedStatusAPITest.java

```diff
@@ -0,0 +1,247 @@
+
+public class ItemDamagedStatusAPITest extends TestBase {
+
+  public static final String TEST_TENANT = "test_tenant";
+
+  @Before
+  public void beforeEach() {
+    StorageTestSuite.deleteAll(TEST_TENANT, ItemDamagedStatusAPI.REFERENCE_TABLE);
+  }
+
+  @Test
+  public void canCreateItemDamagedStatus() {
+
+    ItemDamageStatus status = new ItemDamageStatus()
+      .withName("test item damaged status name")
+      .withSource("local");
+
+    client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT)
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_CREATED.toInt()));
+        JsonObject statusJson = response.getJson();
+        assertThat(statusJson.getString("id"), notNullValue());
+        assertThat(statusJson.getString("name"), is(status.getName()));
+      });
+  }
+
+  @Test
+  public void shouldNotCreateItemDamagedStatusWhenUnprocessableEntity() {
+    JsonObject object = new JsonObject().put("foo", "boo");
+    client.post(itemDamagedStatusesUrl(EMPTY), object, TEST_TENANT)
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(422));
+      });
+  }
+
+  @Test
+  public void shouldNotCreateItemDamagedStatusWithTheSameName()
+    throws MalformedURLException {
+
+    ItemDamageStatus status = new ItemDamageStatus()
+      .withName("test item damaged status name")
+      .withSource("local");
+
+    client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT)
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_CREATED.toInt()));
+      });
+
+    client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT)
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_BAD_REQUEST.toInt()));
+      });
+  }
+
+  @Test
+  public void shouldNotCreateItemDamagedStatusWithTheSameId() {
+
+    ItemDamageStatus status = new ItemDamageStatus()
+      .withName("test item damaged status name")
+      .withSource("local");
+
+    client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT)
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_CREATED.toInt()));
+      })
+      .thenCompose(response -> {
+        status.setId(response.getJson().getString("id"));
+        return client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT);
+      })
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_BAD_REQUEST.toInt()));
+      });
+  }
+
+  @Test
+  public void canGetItemDamagedStatus() {
+
```

Comment:
The other tests do not have a leading blank line, I believe this can be removed

Reply:
Done